interface-definitions/ssh.xml.in

```diff
@@ -206,6 +206,37 @@
             </properties>
             <defaultValue>22</defaultValue>
           </leafNode>
+          <node name="rekey">
+            <properties>
+              <help>SSH session rekey limit</help>
+            </properties>
+            <children>
+              <leafNode name="data">
+                <properties>
+                  <help>Threshold data in Megabytes</help>
```

Comment:
We shouldn't capitalize units. Unless Megabyte is someone's name, it should be lowercase. ;)
Same goes for minutes.

Reply:
Fixed